scripts/demo_cancellation_workflow.py

```
@@ -69,14 +77,16 @@ def demo_workflow():
 
     # Scenario 4: An extreme value job appears
     print("\n4️⃣ EXTREME VALUE job appears: $5,000")
-    should_cancel = manager.cancellation_manager.should_cancel_for_job(5000.0, "job_extreme")
+    should_cancel = manager.cancellation_manager.should_cancel_for_job(
+        5000.0, "job_extreme"
+    )
     print(f"   → Decision: {'Cancel' if should_cancel else 'Keep current'}")
     print(f"   📝 Rationale: Always cancel for jobs > $1,000")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)